src/components/atoms/CalendarEachRow/CalendarEachRow.vue

```diff
@@ -1,10 +1,20 @@
 <template>
-  <p>{{ formatDate(date, 'MM/dd EEEE') }}</p>
+  <p>
+    <span>{{ formatDate(date, 'MM/dd EEEE') }}</span>
+
+    <span v-for="(schedule, index) in schedules" :key="index">
```

Comment:
```suggestion
    <span v-for="(schedule) in schedules" :key="schedule.id">
```
IDなどユニークな値を持ってるならそっちを積極的に使ってもらえればと